### PR TITLE
DR-1081: Use a separate env var for data project in testing situations

### PR DESCRIPTION
**Pull request by marikomedlock**
- Added a separate application property google.singleDataProjectId to specify the data project used by the OneDataLocationSelector class.
- The application property is set to an environment variable GOOGLE_CLOUD_DATA_PROJECT.

- Previously, the OneDataLocationSelector class constructed the data project name by appending "-data" to the environment variable GOOGLE_CLOUD_PROJECT. This is confusing because our development environments no longer live in their own projects. We all use the broad-jade-dev project to run the DR manager. We continue to use separate data projects (e.g. broad-jade-mm-data), but our original projects (e.g. broad-jade-mm) are no longer used.

### CODE REVIEW

**Comment by RoriCremer** [comment]
Excuse me while I go tattoo this on my arm:
```Previously, the OneDataLocationSelector class constructed the data project name by appending "-data" to the environment variable GOOGLE_CLOUD_PROJECT. This is confusing because our development environments no longer live in their own projects. We all use the broad-jade-dev project to run the DR manager. We continue to use separate data projects (e.g. broad-jade-mm-data), but our original projects (e.g. broad-jade-mm) are no longer used.```

**Review by RoriCremer — APPROVED** [review]

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -57,3 +57,4 @@ google.projectCreateTimeoutSeconds=600
 google.coreBillingAccount=00708C-45D19D-27AAFA
 google.parentResourceType=folder
 google.parentResourceId=270278425081
+google.singleDataProjectId=${GOOGLE_CLOUD_DATA_PROJECT}
```
Comment (RoriCremer) [inline src/main/resources/application.properties:60]: Will it ever matter if this is out of sync with the billing account? I am slightly concerned that I could charge the wrong group now that this is corrected. (Which will be my own fault, not this prs, lol)

**Review by ddietterich — APPROVED** [review]
lgtm

**Review by fboulnois — APPROVED** [review]
lgtm

**Pull request merged**